compiler/src/Language/Bond/Codegen/Cpp/Grpc_h.hs

```diff
@@ -263,11 +263,11 @@ inline #{className}::#{proxyName}<TThreadPool>::#{proxyName}(
             Async#{methodName}(::std::make_shared< ::grpc::ClientContext>(), #{bonded methodInput}{request});
         }|]
 
-        privateProxyMethodDecl Function{..} = [lt|const ::grpc::RpcMethod rpcmethod_#{methodName}_;|]
-        privateProxyMethodDecl Event{..} = [lt|const ::grpc::RpcMethod rpcmethod_#{methodName}_;|]
+        privateProxyMethodDecl Function{..} = [lt|const ::grpc::internal::RpcMethod rpcmethod_#{methodName}_;|]
```

Comment:
The addition of the `internal` namespace makes sense here. It's an artifact of [gRPC proposal L8, "Separate internal-only and public sections of the C++ wrapping"](https://github.com/grpc/proposal/blob/master/L8-c%2B%2B-internalization.md). The "internal" parts of gRPC++ are not intended to be used by code using clients or implementing services. The internal parts are OK to be used by things like codegen, which gbc is doing.